### ISSUE TITLE
Implementation of new documentation tooling based on exploration outcomes

### ISSUE THREAD

**Issue by MaxDiz** [medic/medic-docs#198]
Based on the outcome of #196, we may want to implement new documentation tooling. Implementation of the new documentation tooling should reference #194 , #197 , and #202. This ticket is to capture related implementation detail once a pathway is determined.

**Comment by MaxDiz** [comment]
blocked by #196 
Closing. New issue to be opened in CHT-Core when needed
